source/app-development/interactive/form.rst

```diff
@@ -90,13 +90,22 @@ part of the app developer can look like:
 
    # ${HOME}/ondemand/dev/my_app/form.yml
    ---
-   cluster: "owens"
+   cluster: 
+     - "owens"
+     - "pitzer"
    form:
      - bc_queue
      - bc_account
 
 The most commonly used predefined attributes are given as:
 
+cluster
+  This adds a ``select`` to the HTML form that will allow the user to select 
+  what cluster they want to submit the job.
+
+  If there is only one cluster set in ``form.yml``, the select box will not
+  be added and the job will be sent to the specified cluster.
```

Comment:
Leave it as `cluster: "owens"`. Update the description that says cluster specifies the cluster the job will be submitted to. Then explain this can be a list of items and show a snippet below and explain what will happen if that is a list (a dropdown with both will appear). Then explain that you can use glob style match of the clusters, so could do "*" or "cluster*" if you had 2 clusters "cluster1" and "cluster2" defined in the cluster configs, they would both appear in the dropdown to select.

Reply:
Also say this is now an OPTIONAL field as you can now specify cluster in the submit.yml instead.